sdk/communication/AzureCommunication/Source/Authentication/CommunicationUserCredential.swift

```diff
@@ -84,3 +80,20 @@ public typealias TokenRefreshOnCompletion = (String?, Error?) -> Void
         userTokenCredential.token(completionHandler: completionHandler)
     }
 }
+
+@objcMembers public class CommunicationTokenRefreshOptions: NSObject {
```

Comment:
I put this object in `CommunicationUserCredential` but it should be in its own class. 
Should we create a new Utils folder? Cause right now the authentication folder is slowly turning into just a dumping ground for our code and not very well organized

Reply:
Yea agreed, probably best to move this out to its own file.